golang/message-service/Dockerfile

```diff
@@ -1,10 +1,15 @@
-FROM golang:1.16-alpine AS build
+FROM golang:1.18 AS build
```

Comment:
Should we update to 1.19 not it's out?

Reply:
I think I picked 1.18 as "modern enough not to be obsolete, not so modern to cause issues"